packages/astro/src/vite-plugin-astro/index.ts

```diff
@@ -121,16 +125,6 @@ ${err.url}`;
           err.stack = `    at ${id}`;
         }
 
-        // improve esbuild errors
-        if (err.errors && tsResult?.map) {
```

Comment:
This code was trying to tap into Vite’s error overlay for errors. It ended up not being used.